### PR TITLE
updating biolinkml -> linkml per modeling language name change

### PR DESCRIPTION
**Pull request by sierra-moxon**

### CODE REVIEW

**Comment by sierra-moxon** [comment]
@hsolbrig - can you take a look at this 'draft' PR for completeness?  thanks! :)

**Comment by dgarijo** [comment]
@sierra-moxon any updates on this PR?

File: biolink/README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ Biolink
 
 Homepages
 * https://biolink.github.io/biolink-model/ -- Biolink model
-* https://biolink.github.io/biolinkml/ -- Biolink modeling language
+* https://linkml.github.io/ -- Biolink modeling language
```
Comment (cmungall) [inline biolink/README.md:6]: this is fine

**Review by dgarijo — APPROVED** [review]

**Comment by dgarijo** [comment]
I can't merge (it's a draft PR)

**Pull request closed without merging**